SimpleRecorder/SimpleRecorder.csproj

```diff
@@ -136,10 +135,8 @@
     <AppxManifest Include="Package.appxmanifest">
       <SubType>Designer</SubType>
     </AppxManifest>
-    <None Include="SimpleRecorder_TemporaryKey.pfx" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="Package.StoreAssociation.xml" />
```

Comment:
This doesn't seem required? i might be wrong, but in any case it was missing from the repo